hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/KeyValueContainer.java

```diff
@@ -318,6 +326,8 @@ public void close() throws StorageContainerException {
     // It is ok if this operation takes a bit of time.
     // Close container is not expected to be instantaneous.
     compactDB();
```

Comment:
question unrelated to this patch : why do we need to run compact during close? what is the benefit?

Reply:
Good question. I feel we don't have to run compactDB in quasi close or close.